tileview/src/main/java/com/qozix/tileview/graphics/BitmapCleanup.java

```diff
@@ -0,0 +1,12 @@
+package com.qozix.tileview.graphics;
+
+import android.graphics.Bitmap;
+
+/**
+ * This interface represents the "cleanup" operations needed for the Bitmap of a Tile after it is no
```

Comment:
A little remark here : maybe change this line comment with ```"recycle" for a {@link Bitmap} of a {@link Tile} after it is no longer needed for the current display. ```
"cleanup" is the correct for what is done right now, but not necessarily for a user implementation. I admit this also applies for "recycle", but this word is more broad.